src/main/java/gregtech/common/terminal/app/worldprospector/WorldProspectorARApp.java

```diff
@@ -319,6 +392,8 @@ private static void addCluster(BlockPos pos, Map<AxisAlignedBB, Set<BlockPos>> f
     @Override
     public void tickAR(EntityPlayer player) {
         World world = player.world;
+        if (Minecraft.getMinecraft().isGamePaused()) return;
```

Comment:
Does this need to check if `world.isRemote`, or is this method only called on the client?

Reply:
Client side only